### PR TITLE
Made Amount Fraction Currency Specific

### PR DESCRIPTION
**Pull request by OmkarJ13**
[workerB](https://workerb.linearb.io/v2/badge/collaboration-page?magicLinkId=1atmou-)

### CODE REVIEW

**Review by abhishek1234321 — APPROVED** [review]

File: fyle_teams_app/libs/utils.py [hunk 1]
```diff
@@ -48,3 +49,78 @@ def decode_state(state: str) -> Dict:
...
+    # Sign to add at the beginning
+    sign = '-' if amount < 0 else ''
+    amount = abs(amount)
+
+    # Gets the currency precision and symbol for currency specified
+    formatted_currency = format_currency(currency)
+    currency_precision = get_currency_precision(currency)
+
+    # Create a format string and round the amount to currency precision
+    format_string = '{:,.' + str(currency_precision) + 'f}'
```
Comment (jatzz10) [inline fyle_teams_app/libs/utils.py:116]: same comments as slack pr for this file
please check @OmkarJ13

File: fyle_teams_app/ui/cards/notifications.py [hunk 2]
```diff
@@ -191,7 +191,7 @@ def get_report_approval_card(report: Dict, message: str = None, can_approve_repo
                         'facts': [
                             {
                                 'title': 'Amount',
-                                'value': '{} {}'.format(report['currency'], report['amount'])
+                                'value': '{}'.format(utils.get_display_amount(report['amount'], report['currency']))
```
Comment (jatzz10) [inline fyle_teams_app/ui/cards/notifications.py:194]: please use a variable `display_account` -
```
display_account = utils.get_display_amount(report['amount'], report['currency'])
```

File: fyle_teams_app/ui/cards/notifications.py [hunk 3]
```diff
@@ -344,7 +344,7 @@ def get_report_details_card(report: Dict, headline_text: str) -> Dict:
                             },
                             {
                                 'title': 'Amount',
-                                'value': '{} {}'.format(report['currency'], report['amount'])
+                                'value': '{}'.format(utils.get_display_amount(report['amount'], report['currency']))
```
Comment (jatzz10) [inline fyle_teams_app/ui/cards/notifications.py:347]: same as above comment

**Review by jatzz10 — CHANGES_REQUESTED** [review]
@OmkarJ13 few comments
please fix

File: fyle_teams_app/libs/utils.py (hunk 1 above)
Comment (OmkarJ13) [reply fyle_teams_app/libs/utils.py:116]: Done ✅

File: fyle_teams_app/ui/cards/notifications.py (hunk 2 above)
Comment (OmkarJ13) [reply fyle_teams_app/ui/cards/notifications.py]: Done ✅

File: fyle_teams_app/ui/cards/notifications.py (hunk 3 above)
Comment (OmkarJ13) [reply fyle_teams_app/ui/cards/notifications.py]: Done ✅

**Review by jatzz10 — APPROVED** [review]
lgtm
@OmkarJ13 please make sure to test the full flow of teams-app

**Pull request merged**